web/packages/teleport/src/teleportContext.tsx

```diff
@@ -181,6 +181,16 @@ class TeleportContext implements types.Context {
       );
     }
 
+    function hasAccessGraphIntegrationsAccess() {
+      return (
+        userContext.getIntegrationsAccess().list &&
+        userContext.getIntegrationsAccess().read &&
+        userContext.getPluginsAccess().read &&
+        userContext.getDiscoveryConfigAccess().list &&
+        userContext.getDiscoveryConfigAccess().read
+      );
+    }
+
```

Comment:
is there any case where a person may be missing just one of these and then they lose all access?

Reply:
We use all of these to grab the integrations so if they don't have one the page would fail to load

Reply:
roger